nengo_bones/templates/setup.cfg.template

```diff
@@ -67,9 +67,7 @@ max-line-length = 88
 [tool:pytest]
 {% if pytest.addopts %}
 addopts = {%- for opt in pytest.addopts %} {{ opt }} {%- endfor %}
-
```

Comment:
This newline is necessary (otherwise `norecursedirs` ends up on the same line as `addopts`).

Reply:
Right yeah will add it back. I wish Jinja's whitespace handling was better